Gemfile

```diff
@@ -1,5 +1,14 @@
 source 'https://rubygems.org'
 
+gem 'pg'
+gem 'sinatra'
+gem 'sinatra-contrib'
+gem 'capybara'
+gem 'webrick'
+
+gem 'capybara', group: :test
```

Comment:
My rubocop said maybe it was best to not have two versions of cpaybara and rspec in the same gem file as this may cause issues if one gets upgraded and the other doesn't.